pkg/kafka/consumergroup/util.go

```diff
@@ -1,21 +1,23 @@
 package consumergroup
 
 import (
-	"errors"
 	"regexp"
 
-	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/flag"
+	flagutil "github.com/redhat-developer/app-services-cli/pkg/cmdutil/flags"
 	kafkainstanceclient "github.com/redhat-developer/app-services-sdk-go/kafkainstance/apiv1internal/client"
 )
 
 // valid values for consumer group reset offset operaion
 const (
-	AbsoluteOffset  = "absolute"
-	EarliestOffset  = "earliest"
-	TimestampOffset = "timestamp"
-	LatestOffset    = "latest"
+	OffsetAbssolute = "absolute"
+	OffsetEarliest  = "earliest"
+	OffsetTimestamp = "timestamp"
+	OffsetLatest    = "latest"
 )
 
+var ValidOffsets = []string{OffsetAbssolute, OffsetEarliest, OffsetTimestamp, OffsetLatest}
+
 var timestampOffsetRegExp = regexp.MustCompile(`^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}-\d{2}:\d{2})$`)
```

Comment:
It could be better to use an iso8601 validator lib for this:

https://github.com/relvacode/iso8601

Reply:
Move to validators package.